test/e2e/minimatch/minimatch.go

```diff
@@ -1,55 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Lets discuss offline the motivation for this. I think core Open Match initialization is scoped to minimatch.go - and then the other user components initialization are split into their own files (matchfunction.go which in future initializes multiple mmfs needed by tests - and also in future having evaluator.go for evaluator initialization) seems a reasonable way to split files. Why merge all of them? alternately to not confuse them with test cases, you could rename them _setup? (minimatch_setup.go, mmf_setup.go, evaluator_setup.go) and the test files to have _test.go